core/src/main/resources/hudson/widgets/HistoryWidget/entry.jelly

```diff
@@ -34,54 +34,57 @@ THE SOFTWARE.
   <j:if test="${h.isUserTimeZoneOverride()}">
     <i:setTimeZone value="${h.getUserTimeZone()}" />
    </j:if>
-  <tr class="build-row ${transitive} single-line" page-entry-id="${pageEntry.entryId}">
-    <td class="build-row-cell">
-      <div class="pane build-name">
-        <div class="build-icon">
-          <a class="build-status-link" href="${h.getConsoleUrl(build)}" tooltip="${build.iconColor.description} > ${%Console Output}">
-            <l:icon src="symbol-status-${build.iconColor.iconName}" />
-          </a>
-        </div>
-        <a class="model-link inside build-link display-name" update-parent-class=".build-row" href="${link}">${build.displayName}</a>
-      </div>
-      <div class="pane build-details" time="${build.timestamp.time.time}">
-        <j:set var="linkTitleAttr" value=""/> 
-        <j:if test="${!build.building}">
-          <j:set var="linkTitleAttr">${%Took} ${build.durationString}</j:set>
-        </j:if>
-        <a class="model-link inside build-link" href="${link}" update-parent-class=".build-row" tooltip="${linkTitleAttr}">
-          <i:formatDate value="${build.timestamp.time}" type="both" dateStyle="medium" timeStyle="short" /> ${h.getUserTimeZonePostfix(build.timestamp.time)}
-        </a>
-        <j:if test="${build.building}">
-	  <j:set target="${it.widget}" property="nextBuildNumberToFetch" value="${build.number}"/>
-          <t:buildProgressBar build="${build}"/>
-        </j:if>
-      </div>
-      <div class="pane build-controls">
-        <div class="middle-align build-badge">
-          <j:set var="badges" value="${build.badgeActions}"/>
-          <j:if test="${!empty(badges)}">
-            <st:nbsp/>
-            <j:forEach var="badge" items="${badges}">
-              <st:include it="${badge}" page="badge.jelly" />
-            </j:forEach>
+
+  <div class="app-builds-container__item" page-entry-id="${pageEntry.entryId}">
+    <a class="app-builds-container__item__icon"
+       href="${h.getConsoleUrl(build)}"
+       tooltip="${build.iconColor.description}">
```

Comment:
can you add `data-tooltip-append-to-parent="true"` so the tooltips are closed on refresh

Reply:
Done - thanks!